OpenRA.StyleCheck/Program.cs

```diff
@@ -1,8 +1,11 @@
 #region Copyright & License Information
 /*
  * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
- * This file is part of OpenRA, and is made available under the terms of the MS-PL license.
- * For more information, see https://opensource.org/licenses/MS-PL
+ * This file is part of OpenRA, which is free software. It is made
```

Comment:
The Stylecheck project explicitly licensed MS-PL-only to avoid license compatibility issues.  Please drop this change.